next.config.js

```diff
@@ -118,7 +118,39 @@ const nextConfig = {
                 source: '/react-native/v2/features/hls-streaming',
                 destination: '/react-native/v2/features/hls',
                 permanent: false
-            }
+            },
+            {
+                // https://www.markhneedham.com/blog/2022/07/27/vercel-redirect-wildcards-nested-paths/
+                source: '/javascript/v2/guides:path(.*)',
```

Comment:
Needs /:path

Reply:
works without it also? is it not working for you?

Reply:
tested on preview also. can you confirm if it's indeed an issue?